pom.xml

```diff
@@ -2645,7 +2647,18 @@
       <properties>
         <scala.version>2.11.8</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
+        <jline.version>2.12.1</jline.version>
+        <jline.groupid>jline</jline.groupid>
       </properties>
+      <dependencyManagement>
```

Comment:
I just aligned it with the profile for scala 2.10.x, And from my point of view it works. Includes the right Jline version in the delivered tarball.